admin/tests/unit/controllers/forms-xml.js

```diff
@@ -1,12 +1,15 @@
 describe('FormsXmlCtrl controller', () => {
   'use strict';
 
+  const NOW = moment('2000-01-15').valueOf();
```

Comment:
This constant is used only by one test, please define it in the nearest scope instead  
I think the best place for it would be in `describe('saves the doc', () => { ... })`

Reply:
Good call! Done.